src/Rule/InChoice.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Morebec\Validator\Rule;
+
+
+use Morebec\Validator\ValidationRuleInterface;
+
+class InChoice implements ValidationRuleInterface
```

Comment:
Thinking of it, maybe IsInChoice would be a better name, as it follows the other rules:
- Positives: `IsXYZ`
- Negative: `NotXYZ`